frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -109,28 +129,77 @@
 import { Vue, Component, Prop } from 'vue-property-decorator';
 import omegaup_Markdown from '../Markdown.vue';
 import T from '../../lang';
-
+import * as time from '../../time';
+import DatePicker from '../DatePicker.vue';
+import * as ui from '../../ui';
 @Component({
   components: {
     'omegaup-markdown': omegaup_Markdown,
+    'omegaup-datepicker': DatePicker,
   },
 })
 export default class Signup extends Vue {
   @Prop() validateRecaptcha!: boolean;
-
   T = T;
   username: string = '';
-  email: string = '';
+  email: null | string = null;
+  parentEmail: null | string = null;
   password: string = '';
   passwordConfirmation: string = '';
   recaptchaResponse: string = '';
+  birthDate: null | Date = null;
+  privacyPolicyAccepted = false;
+
+  get loginEmailDescriptionText(): string {
+    if (!this.isU13) {
+      return T.loginEmail;
+    }
+    return !this.isU13 ? T.loginEmail : T.loginEmailParent;
+  }
```

Comment:
```suggestion
  }

```

nit: one empty line between functions.